rbd/functional/test_rbd_s3_upload.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+import logging
+import os
+import sys
+
+import boto3
+
+log = logging.getLogger()
+
+# sys.path.append(os.path.abspath(os.path.join(__file__, "../../../")))
```

Comment:
Please remove this commented code as it's not used, `os` `sys` import looks redundant as it's not used anywhere in the code.

Reply:
Done.